### PR TITLE
Simplify coverage workflow

### PR DESCRIPTION
**Pull request by abarth**
We now download the base coverage file automatically.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter_tools/lib/src/base/net.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  printTrace('Received response statusCode=${response.statusCode}');
+  if (response.statusCode != 200)
+    throw new Exception(response.reasonPhrase);
+
+  BytesBuilder responseBody = new BytesBuilder(copy: false);
+  await for (List<int> chunk in response)
+    responseBody.add(chunk);
+
+  return responseBody.takeBytes();
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/base/net.dart]: You can just do `return await response.toList()`

File: packages/flutter_tools/lib/src/commands/update_packages.dart [hunk 2]
```diff
@@ -44,9 +61,10 @@ class UpdatePackagesCommand extends FlutterCommand {
         count++;
       }
 
-      double seconds = timer.elapsedMilliseconds / 1000.0;
-      printStatus('\nRan \'pub\' $count time${count == 1 ? "" : "s"} in ${seconds.toStringAsFixed(1)}s.');
+      await _downloadCoverageData();
 
+      final double seconds = timer.elapsedMilliseconds / 1000.0;
+      printStatus('\nRan \'pub\' $count time${count == 1 ? "" : "s"} in ${seconds.toStringAsFixed(1)}s.');
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/update_packages.dart]: ran pub n times _and fetched coverage data_ in Ts

**Comment by Hixie** [comment]
LGTM

but travis is angry

**Pull request merged**